src/pages/ExamsPage/components/ExamSelection.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+  MenuItem, SearchField,
+  SelectMenu,
+} from '@edx/paragon';
+import PropTypes from 'prop-types';
+import { useUpdateCurrentExamIndex } from '../hooks';
+
+// todo: implement substring search
+
+const ExamSelection = ({ exams }) => {
+  const { updateSelectedExam } = useUpdateCurrentExamIndex();
```

Comment:
Currently getting an error that this is not a function.

Reply:
I think this might be because useUpdateCurrentExamIndex is returning a function instead of an object containing one or more functions and it's the destructuring that's erroring here.